config.default.py

```diff
@@ -3,6 +3,8 @@
     'port': 5000
 }
 
+canonical_host = 'http://127.0.0.1:5000'
```

Comment:
Where is this used?

Reply:
https://github.com/PPCG-v2/PPCG-v2/blob/style-overhaul/templates/post/view.html

Reply:
Next question: why is this used? Hardcoding a URL (especially as an IP address) is counter-intuitive. Use `request.url` instead - it won't break if we use multiple subdomains.